shared/src/Unison/Node/Builtin.hs

```diff
@@ -105,6 +105,11 @@ makeBuiltins whnf =
            op [a] = pure $ Term.remote (Remote.Step (Remote.Local (Remote.Pure a)))
            op _ = fail "unpossible"
        in (r, Just (I.Primop 1 op), remoteSignatureOf "Remote.pure", prefix "pure")
+     , let r = R.Builtin "Remote.map"
```

Comment:
@refried  Isn't that what I did here?

I probably could have used unsafeParseTerm, if that's what you're getting at.